blocks/results-list-block/features/results-list/results/result-item.jsx

```diff
@@ -81,7 +81,8 @@ const ResultItem = React.memo(
 			const imageURL = imageANSToImageSrc(getImageFromANS(element)) || null;
 			const auth = getImageFromANS(element)?.auth || {};
 			const { searchableField } = useEditableContent();
-			const hasAuthors = showByline ? credits?.by && credits?.by.length : null;
+			const hasSomeAuthorName = credits?.by.some((author) => author.name && author?.name !== "");
+			const hasAuthors = showByline ? credits?.by && credits?.by.length && hasSomeAuthorName : null;
```

Comment:
The nullish operator in the first line is all you should need here.  The .some() function will throw an error if it's undefined so we should check before this or nullish chain this as well.  That alone should actually invalidate the old test since it was only looking at the length of the `by` array and this will also do that by checking the author name as well:
```suggestion
			const hasAuthors = showByline ? credits?.by?.some((author) => author?.name !== "") : null;
```

Reply:
@nschubach thank you for you feedback on this, I have pushed the suggested change, also I have confirmed with Tarer and this one was the only block with the bug related with the authors name.